### ISSUE TITLE
[A11y]: Screen reader does not properly read chat window content

### ISSUE THREAD

**Issue by SrfHead** [carbon-design-system/carbon-ai-chat#2322]
### Browser

Firefox

### Operating system

Windows

### Package version

1.15

### React version (if applicable)

18.2.0

### Automated testing tool and ruleset

Latest

### Assistive technology

Windows Narrator

### Description

Using the Windows Narrator screen reader does not give accept results. There is a written notification that describes navigating with Tab/Arrow keys or F6, which is only helpful if you know it's there or are scanning around due to lack of response.
String changes that give a visual queue that the AI response is taking time are not read out. This means there is no audio feedback for when a query has completed.
Even with the F6 switch between input and response, this only works reliably if you wait until you see the response return. If you switch to the chat window before the response returns, you usually end up on a time stamp when you switch back and forth.
Using Tab and arrow keys to navigate the chat window will either focus on time stamps or individual characters. In very few occasions, the chat messages actually read out. 

### WCAG 2.2 violation

503.4.2

### Reproduction / example

N/A

### Steps to reproduce

Enable a screen reader. Attempt to use the AI agent.

### Suggested severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems

**Comment by davidmenendez** [comment]
hey @SrfHead thanks for bringing this up. i noticed this same behavior while working on another issue and have opened a PR that i believe will resolve this https://github.com/carbon-design-system/carbon-ai-chat/pull/2347

can you check the [demo app in the deploy preview](https://deploy-preview-2347--carbon-ai-chat-demo.netlify.app/) and verify if the behavior persists? i was able to have the content properly read to me using mac voice over

**Comment by SrfHead** [comment]
@davidmenendez Hi David, thank you for the update. It definitely works better but I found that it stops responding after a few repeat messages. I don't know if this is a demo/narrator/code issue, but entering "Help" multiple times, the narrator stops reading the response.
If you are confident that the screen reader does not get locked in other circumstances, this should resolve the issue I saw.

**Comment by davidmenendez** [comment]
i think we can close this issue specifically, but i'd like to follow back up. we've worked with Raghu to audit ai chat for our ACS record and we closed out all his team's issues, but it has been a while and i don't know if they used Windows Narrator. So, i think it would be super helpful for us if you could document anything you're experiencing via screen recordings and code replications.

i feel confident that it's working, at least with voice over and the tools i have given my experience with them, but i'm definitely not the expert haha. so, yeah, lets close this out and if you want to work with us to give us a more in depth analysis of what you're seeing with your tools that would be super helpful and lets document it in a new issue that's more specific to any issues we're encountering 👍

**Comment by SrfHead** [comment]
I'm happy for this to close. We are slow to pick up new versions but when this is included, I'll do some further testing. Do you know what level this will likely be shipped in?

**Comment by davidmenendez** [comment]
i'll make sure it gets out in our next release 👍
